Source/CompProperties_SpinalMount.cs

```diff
@@ -0,0 +1,19 @@
+﻿using SaveOurShip2;
+using Verse;
+
+namespace TheCafFiend
+{
+    public class CompProperties_SpinalEngineMount : CompProps_SpinalMount
+    {
+        public int supportWeight = 0; // additional "engineMass" for the object
```

Comment:
The base mod SOS2 implemented "engineMass" which is 30 times (yep, thirty) more impactful to the thrust:weight ratio of your ship, and then feeds it through a math transform. The idea of this is that thrust is basically an abstraction of both raw thrust *and* maneuverability, and mass would reduce both of these, combined with a design decision to disincentivize spamming low-tier engines to try to move your ship around fast enough. I get how they got there, I can't think of a better approach off the top of my head, but I don't love it either. So, it has to be kept separate from the regular mass (which SOS2 adds all by itself, no problem) and the "support" mass there is because in order for things to work correctly on their end after the change, the much easier approach is to add the mass of the supporting infra directly to the engine (Since the exhaust part is correctly registered as an engine itself, SOS2 already adds the special additional weight for that part (it's just a multiplier on the total tiles occupied), but not for the fuel infra)